CLI.md

```diff
@@ -222,7 +222,9 @@ Open (i.e. activate) a new multi-owner chain deriving the UID from an existing o
 * `--from <CHAIN_ID>` — Chain ID (must be one of our chains)
 * `--super-owner-public-keys <SUPER_OWNER_PUBLIC_KEYS>` — Public keys of the new super owners
 * `--owner-public-keys <OWNER_PUBLIC_KEYS>` — Public keys of the new regular owners
-* `--owner-weights <OWNER_WEIGHTS>` — Weights for the new owners
+* `--owner-weights <OWNER_WEIGHTS>` — Weights for the new owners.
+
+   If they are specified there must be exactly one weight for each owner. If no weights are given, every owner will have weight 100.
```

Comment:
Could you move these documentation changes to a separate PR?

Reply:
Hmm, I didn't wanna add this MD file, too.
But, the CI complain made me rebuild the `CLI.md` file. 
https://github.com/linera-io/linera-protocol/actions/runs/10306509270/job/28529761766

Should I roll back this change regardless of the CI complaints?

Reply:
It's probably because you upgraded `clap` (and lots of other crates).

Let's revert the `Cargo.toml` changes (other than what's necessary to add serde) and this one, since they are not related to this change.